win32ss/user/ntuser/defwnd.c

```diff
@@ -929,12 +929,17 @@ IntDefWindowProc(
       case WM_MOUSEACTIVATE:
          if (Wnd->style & WS_CHILD)
          {
-             LONG Ret;
```

Comment:
Leave it here please, we don't tend to follow "smallest scope"

Reply:
Done. (Too bad?)